lib/grp.gd

```diff
@@ -4528,6 +4528,35 @@ DeclareGlobalFunction("GroupEnumeratorByClosure");
 DeclareOperation( "LowIndexSubgroups",
     [ IsGroup, IsPosInt ] );
 
+#############################################################################
+##
+#F  NormalizerViaRadical(<G>,<S>)
+##
+##  <#GAPDoc Label="NormalizerViaRadical">
+##  <ManSection>
+##  <Func Name="NormalizerViaRadical" Arg='G,S'/>
+##
+##  <Description>
+##  This function implements a particular approach, following the
+##  SolvableRadical paradigm, for calculating the
+##  normalizer of a subgroup <A>S</A> in <A>G</A>. It is at the moment
+##  provided only as a separate function, and not as method for the operation
+##  <C>Normalizer</C>, as it can often be slower than other built-in routines.
+##  In certain hard cases (non-solvable groups with nontrivial radical), however
+##  its performance is substantially superior.  The function thus provided as a
+##  non-automated tool for advanced users.
+##  <Example><![CDATA[
+##  gap> g:=TransitiveGroup(30,2030);;
+##  gap> s:=SylowSubgroup(g,5);;
+##  gap> Size(NormalizerViaRadical(g,s));
+##  28800
```

Comment:
Sounds good to me.

Reply:
That's OK, I think (though we might want to mention that the given example is not actually one where `NormalizerViaRadical` is faster?)

Reply:
Added a line about example and inserted `is`.